clients/harvester/config.go

```diff
@@ -0,0 +1,13 @@
+package harvester
+
+// The json/yaml config key for the harvester config
+const ConfigurationFileKey = "harvester"
+
+// Config is configuration need to test against a harvester instance
+type Config struct {
```

Comment:
this is the same as rancher [Config](https://github.com/rancher/shepherd/blob/main/clients/rancher/config.go#L7), why not re use the existing one by adding a flag or something as such?

Reply:
had an offline conversation to help with this understanding. lmk if you have any more questions.